policy/resolver.go

```diff
@@ -1034,41 +1034,39 @@ func (cache *policyResolverCache) addCheckJobVariants(ctx context.Context, query
 }
 
 func (cache *policyResolverCache) addDataQueryJob(ctx context.Context, query *explorer.Mquery, ownerJob *ReportingJob) {
-	if len(query.Variants) != 0 {
-		err := cache.addDataQueryVariants(ctx, query, ownerJob)
-		if err != nil {
-			log.Error().Err(err).Str("queryMrn", query.Mrn).Msg("failed to add data query variants")
-		}
-		return
-	}
-
-	uuid := cache.global.relativeChecksum(query.Checksum)
+	uuid := cache.global.relativeChecksum(query.Mrn)
 	queryJob := cache.global.reportingJobsByUUID[uuid]
 
-	// note: the ReportingJob is only a placeholder and is replaced by individual query LLX checksum ReportingJobs
 	if queryJob == nil {
 		queryJob = &ReportingJob{
-			Uuid:       cache.global.relativeChecksum(query.Checksum),
+			Uuid:       uuid,
 			QrId:       query.Mrn,
 			ChildJobs:  map[string]*explorer.Impact{},
 			Datapoints: map[string]bool{},
 			Type:       ReportingJob_DATA_QUERY,
-			// FIXME: DEPRECATED, remove in v10.0 vv
-			DeprecatedV8IsData: true,
```

Comment:
Shouldn't we keep this until v10?

Reply:
Good point, I will add back. I don't think we use this anywhere but it does not hurt to keep it for the time being